api/v1beta1/cluster_types.go

```diff
@@ -122,6 +122,11 @@ type ControlPlaneTopology struct {
 	// +optional
 	NodeDrainTimeout *metav1.Duration `json:"nodeDrainTimeout,omitempty"`
```

Comment:
(unrelated to this file just had to put it somewhere)

Can we please also set the field to something no-opsy in `cluster-with-topology.yaml` to get the happy path covered in e2e as well.